src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl

```diff
@@ -99,7 +99,7 @@ function HydrostaticFreeSurfaceModel(; grid,
                                   tracer_advection = CenteredSecondOrder(),
                                           buoyancy = SeawaterBuoyancy(eltype(grid)),
                                           coriolis = nothing,
-                                      free_surface = ImplicitFreeSurface(gravitational_acceleration = g_Earth),
+                                      free_surface = SplitExplicitFreeSurface(; grid, cfl = 0.7),
```

Comment:
while we are at it we could finally make `grid` a positional arg

Reply:
in summary I'm wondering if we should write this

```julia
free_surface = default_free_surface(grid)
```

and have

```julia
default_free_surface(grid) = SplitExplicitFreeSurface(; grid, cfl = 0.7)
default_free_surface(::RectilinearGrid) = # fft based free surface
```

Don't we need to specify the gravitational acceleration for split-explicit?

Reply:
we could use this constructor
https://github.com/CliMA/Oceananigans.jl/blob/77ee4980a32d39d802fc6c7b2a8ef81b4c6c0c1a/src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl#L92-L101

and add the grid as a `kwarg` of `SplitExplicitSettings`

Reply:
ah yes, and what about `cfl`?

Reply:
or probably just remove `settings` as a `kwarg` but do something more on this line 
```julia
 function SplitExplicitFreeSurface(grid; gravitational_acceleration = g_Earth, kwargs...)
     settings = SplitExplicitSettings(eltype(grid); grid, gravitational_acceleration, kwargs...)) 
  
     Nz = size(grid, 3) 
     η  = ZFaceField(grid, indices = (:, :, Nz+1)) 
     gravitational_acceleration = convert(eltype(grid), gravitational_acceleration) 
  
     return SplitExplicitFreeSurface(η, SplitExplicitState(grid, settings.timestepper), SplitExplicitAuxiliaryFields(grid), 
            gravitational_acceleration, settings) 
 end 
```

Reply:
(plus also making the state and auxiliary fields on new lines)

Reply:
I think I did all of these suggestions